.github/workflows/e2e.yml

```diff
@@ -23,7 +23,7 @@ on:
   workflow_dispatch:
 
 jobs:
-  build:
+  secretsmanager:
```

Comment:
why a separate job is needed??

Reply:
this is required to set env variable on `build` job level. If we add secret manager step under `build` job then we will need to set env variables at 2 steps (run all tests and run required tests) separately